changelog.json

```
@@ -1,5 +1,53 @@
 {
   "releases": {
+    "1.7.1": [
+      "[New] You can now choose to bring your changes with you to a new branch or stash them on the current branch when switching branches - #6107",
+      "[New] Rebase your current branch onto another branch using a guided flow - #5953",
+      "[New] Repository list entries grouped by owner, and recent repositories listed at top - #6923 #7132",
+      "[New] Suggested next steps now includes suggestion to create a pull request after publishing a branch - #7505",
+      "[Added] .resx syntax highlighting - #7235. Thanks @say25!",
+      "[Added] \"Exit\" menu item now has accelerator and access key - #6507. Thanks @AndreiMaga!",
+      "[Added] Help menu entry to view documentation about keyboard shortcuts  - #7184",
+      "[Added] \"Discard all changes\" action under Branch menu - #7394. Thanks @ahuth!",
+      "[Fixed] \"Esc\" key does not close Repository or Branch list - #7177. Thanks @roottool!",
+      "[Fixed] Attempting to revert commits not on current branch results in an error - #6300. Thanks @msftrncs!",
+      "[Fixed] Clicking \"Undo\" doesn't repopulate summary in commit form - #6390. Thanks @humphd!",
+      "[Fixed] Crash when unable to create log file on disk - #7096",
+      "[Fixed] Emoji rendering in app when account name has special characters - #6909",
+      "[Fixed] Error when selecting diff text while diff is updating - #7131",
+      "[Fixed] Files staged outside Desktop for deletion are incorrectly marked as modified after committing - #4133",
+      "[Fixed] Horizontal scroll bar appears unnecessarily when switching branches - #7212",
+      "[Fixed] Icon accessibility labels fail when multiple icons are visible at the same time - #7174",
+      "[Fixed] Incorrectly encoding URLs affects issue filtering - #7506",
+      "[Fixed] License templates do not end with newline character - #6999",
+      "[Fixed] Conflicts banners do not hide after aborting operation outside Desktop - #7046",
+      "[Fixed] Missing tooltips for change indicators in the sidebar - #7174",
+      "[Fixed] Mistaken classification of all crashes being related to launch - #7126",
+      "[Fixed] Unable to switch keyboard layout and retain keyboard focus while using commit form - #6366. Thanks @AndreiMaga!",
+      "[Fixed] Prevent console errors due to underlying component unmounts - #6970",
+      "[Fixed] Menus disabled by activity in inactive repositories - #6313",
+      "[Fixed] Race condition with Git remote lookup may cause push to incorrect remote - #6986",
+      "[Fixed] Restore GitHub Desktop to main screen if external monitor removed - #7418 #2107. Thanks @say25!",
+      "[Fixed] Tab Bar focus ring outlines clip into other elements - #5802. Thanks @Daniel-McCarthy!",
+      "[Improved] \"Automatically Switch Theme\" on macOS does not check theme on launch - #7116. Thanks @say25!",
```

If this is in "improved" I think we should say what it now does. If we have it under "Fixed" it seems reasonable to say what it used to not do.
```suggestion
      "[Improved] \"Automatically Switch Theme\" on macOS checks theme on launch - #7116. Thanks @say25!",
```